### ISSUE TITLE
Representation of Image in EVA

### ISSUE THREAD

**Issue by vateye** [baaivision/EVA#15]
Hi, the EVA model is trained with image tokens, and you use the average pooling for image representation during fine-tuning. I think the CLS token is not well-learned. But for EVA-CLIP, it initialized from the EVA model whose CLS token is not well-learned and uses CLS for pre-training. I am wondering why not using the average pooling for image representation and align with the text?

**Comment by Yuxin-CV** [comment]
This is more about a performance-driven choice. 

The MIM pre-trained EVA usually achieves higher accuracy w/ GAP feature in e2e fine-tuning. This is also ture for other MIM pre-trained ViTs, _e.g._, BEiT, MAE, _etc_.    

For EVA-CLIP, we find using GAP feature achieves sub-optimal performance than using `cls_token`.

During MIM pre-training, although there is no direct supervision on `cls_token`, we believe the `cls_token` indeed represents some kinds of image-level / global content, _e.g._, MAE also uses `cls_token` for linear probing. We also tried with GAP feature for MAE linear probing, the performance is not very satisfactory.

**Comment by vateye** [comment]
OK, thanks for your patient explanation.
